.travis.yml

```diff
@@ -44,7 +44,6 @@ addons:
 install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
-  - sudo apt-get install --yes -qq zip
```

Comment:
i think we can remove the  `jobs` section in travis.yaml above

Reply:
Done.